packages/python/accern_data/accern_data.py

```diff
@@ -217,6 +241,16 @@ def get_current_date(
         raise NotImplementedError()
 
     def get_path(self, is_by_day: bool) -> str:
+        """
+        Generates file path.
+
+        Parameters:
+            is_by_day: Flag to determine whether or not file is divided on the
+            basis of dates.
+
+        Returns:
+            File path.
+        """
         # NOTE: pattern can be None or "" only for csv_date & json.
         # In case of None, filenames would be <date>.csv or <date>.json.
         # In case of empty string (""), filenames would be -<date>.csv or
```

Comment:
this is no longer the case, right? it should be date.csv or date.json in either case

Reply:
no we are still having both `pattern-date.csv` & `date.csv`. same for json